<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWDialog</class>
 <widget class="QDialog" name="QWDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>422</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog by wy</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="chkBoxBold">
      <property name="text">
       <string>Bold</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkBoxItalic">
      <property name="text">
       <string>Italic</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkBoxUnder">
      <property name="text">
       <string>Underline</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>chkBoxItalic</zorder>
   <zorder>chkBoxBold</zorder>
   <zorder>chkBoxUnder</zorder>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="rBtnBlue">
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rBtnBlack">
      <property name="text">
       <string>Black</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rBtnRed">
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>quxiao</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>tuichu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>que ding</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>201</width>
     <height>70</height>
    </rect>
   </property>
   <property name="plainText">
    <string>hello world !   
 it's my  fisrt demo</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
